driver-sync/src/test/functional/com/mongodb/client/JsonPoweredCrudTestHelper.java

```diff
@@ -369,6 +369,27 @@ BsonDocument getDatabaseListCollectionsResult(final BsonDocument databaseOptions
         return toResult(iterable);
     }
 
+    BsonDocument getCreateIndexResult(final BsonDocument collectionOptions, final BsonDocument arguments,
```

Comment:
This helper did not have support for the `createIndex` or `dropIndex` commands, which are needed for the new write concern tests. I have added them here.